tc/core/cuda/cuda_mapping_options.cc

```diff
@@ -318,7 +323,8 @@ CudaMappingOptions CudaMappingOptions::makeUnmappedMappingOptions() {
       .useSharedMemory(false)
       .usePrivateMemory(false)
       .unrollCopyShared(false)
-      .useReadOnlyCache(false);
+      .useReadOnlyCache(false)
+      .timeout(FLAGS_timeout);
```

Comment:
I don't see a good justification to adding debug type information to the protos, I would kill this with fire.